docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -1214,14 +1214,40 @@ If you work with a Quarkus OIDC `web-app` application, then the Quarkus OIDC cod
 To use the refresh token, you should carefully configure the session cookie age.
 The session age should be longer than the ID token lifespan and close to or equal to the refresh token lifespan.
 
-You calculate the session age by adding the lifespan value of the current ID token and the values of the `quarkus.oidc.authentication.session-age-extension` and `quarkus.oidc.token.lifespan-grace` properties.
+You calculate the session cookie age by adding the lifespan value of the current ID token and the values of the `quarkus.oidc.authentication.session-age-extension` and `quarkus.oidc.token.lifespan-grace` properties.
 
 [TIP]
 ====
-You use only the `quarkus.oidc.authentication.session-age-extension` property to significantly extend the session lifespan, if required.
 You use the `quarkus.oidc.token.lifespan-grace` property only to consider some small clock skews.
 ====
 
+[TIP]
+====
+You use the `quarkus.oidc.authentication.session-age-extension` property to significantly extend the session lifespan in order to
+make even expired ID token available to the application.
```

Comment:
I'm not sure what this suggestion proposes ?

Reply:
I Just applied the one sentence-per line rule. The text I am proposing to delete here was moved to a previous row.
Let me double-check that.

Reply:
Yeah, I reformulate it so that it is good to use on a higher row. Then, I deleted the portion of the original text that was no longer needed.
![image](https://github.com/user-attachments/assets/66ed2ba3-f47a-4b0b-97cc-3ee6baaa6d1b)

Reply:
The text from the line 1227 is on the row 1226 now.